exercises/src/test/scala/fpinscala/datastructures/ListProps.scala

```diff
@@ -0,0 +1,151 @@
+package fpinscala.datastructures
+
+import fpinscala.datastructures.List.*
+import fpinscala.datastructures.Tree.*
+import org.scalacheck.*
+import org.scalacheck.Gen.*
+import org.scalacheck.Prop.{forAll, propBoolean}
+
+import scala.language.adhocExtensions
```

Comment:
Is this import needed?

Reply:
Yes, it is if `org.scalacheck.Properties` is needed.
But I will try to use the testing framework from part 2 instead of Scalacheck.